tests/LocalFileSourceTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace ConfigCat\Tests;
+
+use ConfigCat\ConfigCatClient;
+use PHPUnit\Framework\TestCase;
+use InvalidArgumentException;
+
+class LocalFileSourceTest extends TestCase
+{
+    /**
+     * @expectedException InvalidArgumentException
+     */
+    public function testWithNonExistingFile() {
+        new ConfigCatClient("key", [
+            'file-source' => "non-existing",
+        ]);
+    }
+
+    public function testWithFile() {
```

Comment:
GIGA megnyugtató lenne ha lenne leaglább 1 teszteset rule-lal is.

Reply:
Adtam hozza egy errol szolo test case-t